frontend/public/components/persistent-volume.jsx

```diff
@@ -1,26 +1,50 @@
 import * as React from 'react';
 
 import { ColHead, DetailsPage, List, ListHeader, ListPage } from './factory';
-import { Kebab, LabelList, navFactory, ResourceKebab, SectionHeading, ResourceLink, ResourceSummary, Timestamp } from './utils';
+import { Kebab, LabelList, navFactory, ResourceKebab, SectionHeading, ResourceLink, ResourceSummary, Timestamp, StatusIcon } from './utils';
+
+const pvPhase = pv => pv.status.phase;
```

Comment:
I'd inline this if it's only used in the one place. It doesn't save you much if anything